ticketbai.go

```diff
@@ -123,6 +123,10 @@ func New(software *Software, zone l10n.Code, opts ...Option) (*Client, error) {
 		opt(c)
 	}
 
+	if c.cert == nil {
+		return c, nil
```

Comment:
Should it not raise an error instead?

Reply:
It depends on whether a certificate should be mandatory to convert a TicketBAI document. With these changes, if a user wants to convert a document with without a certificate it can (with empty signature and fingerprint).

If a user attempts to send a document from the CLI without a certificate an error is raised when attempting to load the certificate. If a user attempts to fingerprint the document it will do so with no problem, but if it attempts to sign it returns `{"error":"signing: cannot sign without a certificate"}`.

The way I see if we have a Convert() function we want to make accessible (it even has its CLI command), it should not require a certificate, especially because the result is not signed.